src/react-components/media-browser.js

```diff
@@ -212,9 +212,8 @@ class MediaBrowser extends Component {
   };
 
   showCustomMediaDialog = source => {
-    const isSceneApiType = source === "scene_listings" || source === "scenes";
-    // Note: The apiSource for avatars *is* actually singular. We should probably fix this on the backend.
```

Comment:
Were these the back-end change that needed to happen? https://github.com/mozilla/reticulum/pull/179

Reply:
Never mind about this; You cleared it up for me in person. Maybe check in with gfodor about why he wrote this to inspect the search results' types rather than the urlSource.

Reply:
Greg preferred apiSource over urlSource since urlSource was more of a UI implementation detail where as apiSource was theoretically more concrete and less likely to change. However, it seems urlSource is now pretty solidified, but it would be nice to rely on apiSource if our REST responses were consistent in failure cases.